internal/pkg/manifest/storage_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package manifest
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+type testVolume struct {
+	EFS *EFSConfigOrID `yaml:"efs"`
+}
+
+func Test_UnmarshalEFS(t *testing.T) {
```

Comment:
Should we include a few more tests that cover error cases here? For example, in the case of 
```
efs: copilot
   id: managed
```
or 
```
efs: copilot
   id: random_id
```
etc.

Reply:
These edge cases will go away in the next refactor PR; I vote to leave this as is if that's okay.